tests/jquery.flot-relativeTimeAxis.Test.js

```diff
@@ -54,50 +54,63 @@ describe('A Flot chart with relative time axes', function () {
     });
 
     it('shows time bigger than 1 second correctly', function () {
-        plot = createPlotWithRelativeTimeAxis(placeholder, [[[1, 1], [2, 2]]]);
+        plot = createPlotWithRelativeTimeAxis(placeholder, [[[0, 0], [1, 1], [2, 2]]]);
```

Comment:
The concern of the tests was the way time is displayed. Since a relative time axis should always start from 00:00:00, they needed to be modified to still be able to test the way time is displayed in different cases.